src/electionguard/election.py

```diff
@@ -55,6 +66,9 @@ class CiphertextElectionContext:
     extended_data: Optional[Dict[str, str]]
     """Data to allow extending the context for special cases."""
 
+    configuration: Configuration = field(default_factory=Configuration)
```

Comment:
Will we need to change any of our sample data or tests with this new field?

Reply:
Yes, we will need to change all the sample data. We can do this post 1.4 being release though instead of chasing our tails.